spec/javascript/packages/document-capture/components/acuant-capture-spec.jsx

```diff
@@ -1113,6 +1113,25 @@ describe('document-capture/components/acuant-capture', () => {
     });
   });
 
+  context('mobile selfie', () => {
+    it('renders the selfie capture loading div in acuant-capture', async () => {
```

Comment:
I have a hard time believing that this is the only test that needs to be added in this file, but I couldn't come up with any additional tests that made sense to me. Really, I want to test the full "user click the input, then closes" type flow, but that doesn't seem to be possible here.